docs/toml.md

```diff
@@ -56,11 +57,13 @@
 # Backends throttle duration: minimum duration in seconds between 2 events from providers
 # before applying a new configuration. It avoids unnecessary reloads if multiples events
 # are sent in a short amount of time.
+# Can be provided in a format supported by [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration) or as digits
+# only. In the latter case, seconds are assumed.
```

Comment:
`or as digits only` -> `or as raw values (digits)`
`In the latter case, seconds are assumed.` -> `If no units are provided, the value is parsed assuming seconds.`